scripts/migrate-xmpp-to-appserver.py

```diff
@@ -66,6 +66,9 @@ def main():
     parser.add_argument("--managementportal-password", type=str, default="admin",
                         help="The password for the user to authorise with using password grant code.")
 
+    parser.add_argument("--managementportal-secret", type=str, default="",
+                        help="The client secret for ManagementPortalapp client for authorising with password grant code.")
+
```

Comment:
Can you rename this to `managementportal-client-secret` and also add config for `managementportal-client-id` with default `ManagementPortalapp `?